src/components/Shared/LoadingSpinner.js

```diff
@@ -0,0 +1,28 @@
+import styled, { keyframes } from 'styled-components';
+import theme from 'theme';
+
+const rotate360 = keyframes`
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+`;
+
+const LoadingSpinner = styled.div`
+  animation: ${rotate360} 1s linear infinite;
+  transform: translateZ(0);
+
+  border-top: 2px solid ${theme.colors.yellow};
+  border-right: 2px solid ${theme.colors.yellow};
+  border-bottom: 2px solid ${theme.colors.yellow};
+  border-left: 8px solid ${theme.colors.yellow};
```

Comment:
```suggestion
  border: ${theme.colors.yellow} solid;
  border-width: 2px 2px 2px 8px;
```
별 것 아니긴 한데, 이렇게 줄이는 게 어땠을까 싶네

Reply:
```suggestion
  border: 2px solid ${theme.colors.yellow};
  border-bottom-width: 8px;
```
혹은 이렇게?

Reply:
흠 일단 제가 이번 페이지를 만들게 된 이유는
404와 이 로딩스피너의 차이는 조금 다른 이유가 있었는데
1. 404 - 로그인 이후 잘못된 route로 접속할 시
2. 로딩스피터- 로그인 하지 않고 정상적인 route 혹은 잘못된 접근 시
이 두 개를 생각하고 제작하였습니다.

Reply:
잠깐 지나가는 로딩스피너라면, 텍스트를 읽기 힘들 것 같아서요.
"로그인 되지 않은 상태에서 이동하려는 경우에 보여주는 로딩 페이지" 라길래
로그인 되지 않은 채로 로그인이 필요한 화면으로 이동한다면 무한하게 저 화면을 띄워주는거라고 이해했네요 ㅋㅋ

실제로 로딩스피너 연출되는 시간이 꽤 긴 기능이 있나요? 텍스트를 읽고 누를 만 한 정도로

Reply:
이 부분에서는 조금 코알라 로그인 과정에서 생각한 부분인데요. 코알라 로그인 방식이 JWT 방식에 AccessToken을 JS파일내에 저장하고 새로고침이 되면 refresh토큰을 활용해 다시 AccessToken을 저장하는 방식이여서 만약 로그인이 완료되거나 AccessToken을 받아오는 과정이 느린 환경에서는 로딩스피너가 나오는 것이 적절하다고 생각합니다.
그리고 기본적으로 로그인 하지 않고 ~~/keyword 로 접근하면 로딩스피너연출이 길어지게 되므로 로딩스피너가 들어가는것이 자연스럽다고 생각합니다.